tests/test_import_export.py

```diff
@@ -17,6 +17,21 @@
 from tests.data import expected as expected_data, input as input_data
 
 
+def _normalize_xml(dirname):
+    """Removes whitespace from xml files in the given directory."""
+    for dname, _, files in os.walk(dirname):
+        for fname in files:
+            fpath = os.path.join(dname, fname)
+            if not fpath.endswith('.xml'):
+                continue
+            with open(fpath) as f:
+                s = f.read()
```

Comment:
Many of the tests use a utility function `clean_xml` which isn't used in that test since it's using `diff` instead. Can you pass the XML files through that utility function and do the diff on the output?